db/migrations/2025_03_28_181930_create_measurement_techniques.py

```diff
@@ -0,0 +1,34 @@
+import sqlalchemy as sql
+
+
+def up(conn):
+    query = """
+        CREATE TABLE MeasurementTechniques (
+          id            INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+          type          VARCHAR(100) COLLATE utf8mb4_bin NOT NULL,
+          subjectType   VARCHAR(100) COLLATE utf8mb4_bin NOT NULL,
+          units         VARCHAR(100) COLLATE utf8mb4_bin NOT NULL,
+          description   TEXT,
+          includeStd    tinyint(1) NOT NULL DEFAULT '0',
+          studyUniqueId VARCHAR(100) COLLATE utf8mb4_bin NOT NULL,
+          metaboliteIds JSON DEFAULT (JSON_ARRAY()),
+          strainIds     JSON DEFAULT (JSON_ARRAY()),
+
+          createdAt datetime NOT NULL DEFAULT CURRENT_TIMESTAMP,
+          updatedAt datetime NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+
+          CONSTRAINT MeasurementTechniques_studyUniqueId FOREIGN KEY (studyUniqueId)
+              REFERENCES Study (studyUniqueId)
+              ON DELETE CASCADE ON UPDATE CASCADE
+        );
+    """
```

Comment:
Each measurement technique represents a single type of measurement of a single type of subject ("bioreplicate", "strain", or "metabolite"). Individual measurements have an id pointing to the specific technique that they correspond to.